mspray/apps/main/views/performance.py

```diff
@@ -429,22 +427,22 @@ def get_context_data(self, **kwargs):
         ).order_by("spray_date")
         serializer = PerformanceReportSerializer(queryset, many=True)
         totals = {
-            "other": sum([i["other"] for i in serializer.data]),
```

Comment:
I think we should be explicit in exactly what kind of iterable this is.  it should be a generator expression so please include the `(` and `)`.  Explicit is better than implicit: https://www.python.org/dev/peps/pep-0020/  Please do this for the other cases in this module.